pom.xml

```diff
@@ -266,6 +266,28 @@ limitations under the License.
           <ignoreNonCompile>true</ignoreNonCompile>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-enforcer-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>enforce-banned-deps</id>
+            <goals>
+              <goal>enforce</goal>
+            </goals>
+            <configuration>
+              <rules>
+                <!-- ban all log4j 2.x deps with CVEs -->
+                <bannedDependencies>
+                  <excludes>
+                    <exclude>org.apache.logging.log4j:*:[2.0-beta9, 2.14.1]</exclude>
```

Comment:
does this also include pre beta9?

Reply:
should this be log4j-api, not log4j?

Reply:
it's excluding `org.apache.logging.log4j:*` so should include all the artifacts?

Reply:
should it be `log4j*` rather than `log4j:*`? Sorry I'm not sure how this field works, just want to make sure!

Reply:
I think log4j:* should work, example here: https://maven.apache.org/enforcer/enforcer-rules/bannedDependencies.html

I also tested by not updating the version to 2.15 and the build failed